docs/project/2024/victor.istudor/index.md

```diff
@@ -0,0 +1,84 @@
+# Project Name
+A robot that follows a black line on a white background.
+
+:::info 
+
+**Author**: Istudor Victor Mihai \
+**GitHub Project Link**: link_to_github
+
+:::
+
+## Description
+
+This robot will be capable of following a trajectory given by a black line on a white background, with the help of two infrared sensors placed in the front. The robot will be powered by 4 AA batteries that will give current to 4 DC motors connected to a L298N motor driver. 
+
+## Motivation
+
+I chose this project because i started to have a passion for robotics. Being a member of the robotics club within the FILS faculty, i had to build a similar robot for the PoliFest event, but it used an Arduino UNO and the coding part was fairly simple, as it was written in C. As soon as i saw the project rules, i thought that i can give myself a bit of a challenge by building the same robot, but in this case using a Raspberry Pi Pico, having its software written in Rust. 
+
+## Architecture 
+
+<!-- Add here the schematics with the architecture of your project. Make sure to include:
+ - what are the main components (architecture components, not hardware components)
+ - how they connect with each other -->
+
+ ![Robot Architecture](./Schematics.drawio.png)
+
+## Log
+
+<!-- write every week your progress here -->
+
+### Week 6 - 12 May
+
+### Week 7 - 19 May
+
+### Week 20 - 26 May
+
+## Hardware
+
+The "brain" of this robot will be the Raspberry Pi Pico microcontroller. The IR sensors and motor driver will be connected to it. The batteries will be able to power the motor driver and the Pico. All 4 of the 3V-6V DC motors will be hooked-up to the H-bridge.
+
+These are the components that i will use:
+![Components](./Components.jpg)
+
+### Schematics
+
+Place your KiCAD schematics here.
+
+### Bill of Materials
+
+<!-- Fill out this table with all the hardware components that you might need.
+
+The format is 
+```
+| [Device](link://to/device) | This is used ... | [price](link://to/store) |
+
+```
+
+-->
+
+| Device | Usage | Price |
+|--------|--------|-------|
+| [Rapspberry Pi Pico W](https://www.raspberrypi.com/documentation/microcontrollers/raspberry-pi-pico.html) | The microcontroller | [35 RON](https://www.optimusdigital.ro/en/raspberry-pi-boards/12394-raspberry-pi-pico-w.html) |
+| [2x IR Sensor Module] | Used for line detection | [8,78 RON](https://ardushop.ro/ro/electronica/41-modul-senzor-ir-infrarosu-evita-obstacole.html?search_query=infrarosu&results=29) |
```

Comment:
```suggestion
| 2x IR Sensor Module | Used for line detection | [8,78 RON](https://ardushop.ro/ro/electronica/41-modul-senzor-ir-infrarosu-evita-obstacole.html?search_query=infrarosu&results=29) |
```

if there is no link, no brackets are needed. Please do add links to the components.

Reply:
I added links for some of the components and removed unwanted brackets.